llvm/lib/SYCLLowerIR/ComputeModuleRuntimeInfo.cpp

```diff
@@ -155,6 +155,35 @@ std::optional<T> getKernelSingleEltMetadata(const Function &Func,
   return std::nullopt;
 }
 
+PropSetRegTy computeBF16DeviceLibProperties(const Module &M,
+                                            std::string &BF16DeviceLibName) {
+  PropSetRegTy PropSet;
+
+  {
+    for (const auto &F : M.functions()) {
+      if (!F.getName().starts_with("__devicelib_") || F.isDeclaration())
+        continue;
+      if (F.getCallingConv() == CallingConv::SPIR_FUNC) {
+        PropSet.add(PropSetRegTy::SYCL_EXPORTED_SYMBOLS, F.getName(),
+                    /*PropVal=*/true);
+      }
+    }
+  }
+
+  {
+    // Value '0' means fallback version and '1' means native version of
+    // bfloat16 devicelib.
+    unsigned BF16DeviceLibType = 0;
```

Comment:
Do we anticipate more types here? Because if not, renaming the property to something like `isNativeBF16DeviceLib` would make values self-documenting.

Reply:
Hi, @AlexeySachkov 
Done.
Thanks very much.